pulp_ansible/tests/functional/api/collection/v3/test_namespace.py

```diff
@@ -170,38 +170,51 @@ def test_namespace_syncing(
     monitor_task,
     ansible_sync_factory,
     ansible_collection_remote_factory,
+    gen_user,
 ):
     """Test syncing a Collection w/ a Namespace also syncs the Namespace."""
-    # Set up first Repo to have 3 Collections and 3 Namespaces
-    repo1, distro1 = ansible_repo_and_distro_factory()
-    kwargs1 = {"path": distro1.base_path, "distro_base_path": distro1.base_path}
-
-    collections = []
-    namespaces = {}
-    for i in range(3):
-        namespace = random_string()
-        collection, _ = build_and_upload_collection(repo1, config={"namespace": namespace})
-        avatar_path = random_image_factory()
-        task = galaxy_v3_plugin_namespaces_api_client.create(
-            name=namespace, avatar=avatar_path, **kwargs1
-        )
-        result = monitor_task(task.task)
-        collections.append(collection)
-        namespaces[namespace] = result.created_resources[-1]
-
-    # Set up second Repo and sync 2 Collections from first Repo
-    repo2, distro2 = ansible_repo_and_distro_factory()
-    kwargs2 = {"path": distro2.base_path, "distro_base_path": distro2.base_path}
-    collections_string = "\n  - ".join((f"{c.namespace}.{c.name}" for c in collections[0:2]))
-    requirements = f"collections:\n  - {collections_string}"
-    remote = ansible_collection_remote_factory(
-        url=distro1.client_url, requirements_file=requirements
+    user = gen_user(
+        model_roles=[
+            "ansible.ansibledistribution_creator",
+            "ansible.ansiblerepository_creator",
+            "ansible.collectionremote_creator",
+        ]
```

Comment:
Any particular reason you added a user to this test? They should run with admin permissions without specifying the user.

Reply:
I added a user due to a 403 error that I ran into while fixing tests (see [current test failure](https://github.com/pulp/pulp_ansible/actions/runs/4471839876/jobs/7857260320?pr=1338#step:11:2697)). In my local logs I see the following which leads to the missing `path` attribute:
```
pulp_1   | Giving up download_wrapper(...) after 1 tries (aiohttp.client_exceptions.ClientResponseError: 403, message='Forbidden', url=URL('http://localhost:5001/pulp_ansible/galaxy/587105c2-32bf-4f58-ad8b-9b60c1ab6c6b/api/v3/collections/all/'))
pulp_1   | pulp [15fd9c46-dab0-41ea-8173-a07caaa77112]: backoff:ERROR: Giving up download_wrapper(...) after 1 tries (aiohttp.client_exceptions.ClientResponseError: 403, message='Forbidden', url=URL('http://localhost:5001/pulp_ansible/galaxy/587105c2-32bf-4f58-ad8b-9b60c1ab6c6b/api/v3/collections/all/'))
```